reconcile/utils/oc.py

```diff
@@ -940,7 +957,7 @@ def is_kind_supported(self, kind: str) -> bool:
         if "." in kind:
             # self.api_resources contains only the short kind names
             kind = kind.split(".", 1)[0]
-        return self.api_resources and kind in self.api_resources
+        return self.get_api_resources() and kind in self.get_api_resources()
```

Comment:
why not `return kind in self.get_api_resources()` ?

Reply:
Of course... Let me fix that.